flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/plan/nodes/logical/FlinkLogicalWatermarkAssigner.scala

```diff
@@ -35,16 +35,16 @@ class FlinkLogicalWatermarkAssigner(
     cluster: RelOptCluster,
     traits: RelTraitSet,
     input: RelNode,
-    rowtimeField: String,
-    watermarkOffset: Long)
-  extends WatermarkAssigner(cluster, traits, input, rowtimeField, watermarkOffset)
+    rowtimeFieldIndex: Option[Int],
+    watermarkOffset: Option[Long])
```

Comment:
Rename all the other `watermarkOffset` to `watermarkDelay` ?

Reply:
done